test/test_nn.py

```diff
@@ -7379,6 +7379,20 @@ def test_upsamplingNearest2d(self):
         gradcheck(lambda x: F.interpolate(x, 4, mode='nearest'), [input])
         gradgradcheck(lambda x: F.interpolate(x, 4, mode='nearest'), [input])
 
+    def test_upsamplingNearest2d_channels_last(self):
```

Comment:
You can combine both tests by doing something like:
`for memory_format in (A,B):`